fluentd/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM fluent/fluentd:v1.2.1-onbuild
```

Comment:
Why do you not use the image from https://github.com/cynepco3hahue/fluentd-docker-image/blob/master/v1.2?

Reply:
Why can't we still just use the official ones?

Reply:
I prefer using the official image and customize the configuration as needed.